mod/wet4/views/default/forms/login.php

```diff
@@ -25,23 +25,18 @@
 ?>
 <div class="col-sm-12 clearfix mrgn-bttm-lg brdr-bttm">
     <div class="col-sm-4 clearfix" style="padding-top:75px" aria-label="GCconnex Decommission Image">
-        <img class="mrgn-bttm-lg" src="<?php echo $site_url .'mod/wet4/graphics/GCConnex_Decommission_Banner_Image.png'?>" style="max-width:100%" alt="<?php echo elgg_echo('gcx:messaging:alt'); ?>" />
+        <img class="mrgn-bttm-lg" src="<?php echo $site_url .'mod/gcconnex_read_only/graphics/GCConnex_Decom_Final_Banner.png'?>" style="width: 100%; height: auto; padding-right: 20px; margin-top: -25px;" alt="GCConnex Decommissioning Final Banner"<?php echo elgg_echo('gcx:messaging:alt'); ?>" />
```

Comment:
was trying to figure out for a bit why this wasn't working for me, the filename is GCconnex_Decom_Final_Banner.png - second c is lower case.